src/app/src/components/FacilityDetailSidebarContributors.jsx

```diff
@@ -38,18 +49,30 @@ const FacilityDetailSidebarContributors = ({ classes, contributors, push }) => {
     return (
         <div className={classes.item}>
             <Typography className={classes.label}>Contributors</Typography>
-            <Divider className={classes.divider} />
-            {contributors.map(contributor => (
-                <FacilityDetailSidebarDetail
-                    primary={contributor.contributor_name}
-                    secondary={contributor.list_name}
-                    hasAdditionalContent={!!contributor.id}
-                    hideTopDivider
-                    additionalContentCount=""
-                    onClick={() => push(makeProfileRouteLink(contributor.id))}
-                    key={contributor.id}
-                    verified={contributor.is_verified}
-                />
+            {visibleContributors.map(contributor => (
+                <ListItem
```

Comment:
Switched from using the FacilityDetailSidebarDetail here because the structure is different enough now to make it overly complex